doc/policies/conditions.rst

```diff
@@ -30,7 +30,8 @@ consists of five parts:
 Sections
 ~~~~~~~~
 
-privacyIDEA implements three sections ``userinfo``, ``token``, ``tokeninfo`` and ``HTTP Request Headers``.
+privacyIDEA implements three sections ``userinfo``, ``token``, ``tokeninfo``, ``HTTP Request Headers``
```

Comment:
Please change this to "five sections".

Reply:
Will do "the sections" to be future proof.

Reply:
Done in 753c230b